spec/support/rspec_playwright.rb

```diff
@@ -11,7 +11,7 @@ def self.start_browser
     Playwright.create(playwright_cli_executable_path: PLAYWRIGHT_CLI_EXECUTABLE_PATH)
               .playwright
               .chromium
-              .launch(headless:)
+              .launch(headless:, slowMo:)
```

Comment:
Shall we make this Ruby-esque?
```suggestion
              .launch(headless:, slowMo: slow_mo)
```